lib/linters/variable_value.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const util = require('util');
+
+module.exports = {
+    name: 'variableValue',
+    nodeTypes: ['decl'],
+    message: 'The value of [%s] must be a variable.',
+
+    lint: function variableValueLinter (config, node) {
+        if (!config.enabled) {
+            return;
+        }
+
+        if (config.properties.indexOf(node.prop) === -1) {
+            return;
+        }
+
+        if (/^@/i.test(node.value)) {
```

Comment:
Could you add a comment explaining this line, i.e.
`// It's a variable, so it's allowed`